get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -97,174 +147,111 @@ m|:help keys
 |Toggle Editor to Full Screen
 |===
 
-== Built in Guides
-
-If you want to learn more first, click on any of the helpful links shown after startup: these are quick guides that introduce the different concepts.
-You find more helpful links in the left sidebar in the "Information" tab (icon:book[]).
-
-|===
-| Intro | a guided tour of Neo4j browser | `:play intro`
-| Concepts | GraphDB 101 | `:play concepts`
-| Cypher | query language  introduction |`:play cypher`
-| The Movie Graph | a mini graph model with use-cases | `:play movie graph`
-| The Northwind Database | the classic demo database with import instructions & use-case queries | `:play northwind graph`
-| Custom Guides | starting with Neo4j 2.3 you can use `:play <url>` to play a custom guide | `:play http://guides.neo4j.com/graphgist/restaurant_recommendation.html` 
-|===
-
-Import our sample movie graph by entering `:play movie graph`.
-
-On the second slide click the large Cypher +CREATE ...+ statement, then hit the triangular `Run` button.
+== Query and Command-Line Tips
 
-After a few seconds the data is imported, and you'll see a subset of the movie data rendered as a graph.
+You can remove all accumulated output frames with `:clear`. The 'X' button at the top right of each pane removes that frame and aborts a (long-)running statement.
+The maximum number of frames that are kept is configurable in the Browser Settings from the left-side menu.
```

Comment:
mention pinning of frames (e.g. guides) so they stay on top?
perhaps best in the guides section.

Reply:
I actually just discovered this functionality yesterday! 😆  I'll add it.